src/components/breadcrumb/examples/basic.js

```diff
@@ -0,0 +1,27 @@
+/*
+Display breadcrumbs for a page.
+All docs > Site > Subsite > Section > Page title
+*/
+import React from 'react';
+import Breadcrumb from '../breadcrumb';
+
+export default class Basic extends React.Component {
+  render() {
+    return (
+      <Breadcrumb
+        currentPage={{
+          title: 'Display a map',
+          path: '/mapbox-gl-js/examples/display-a-map/'
```

Comment:
Do you think it's important that the current page's path is consistent with the `site/subsite/section` path? I was wondering if we might want to add some kind of validation utility but maybe that would be too restrictive.

Reply:
> Do you think it's important that the current page's path is consistent with the site/subsite/section path?

Not sure I'm following, can you provide an example of what this would look like?

Reply:
Sorry, that wasn't worded very well. I was thinking about a scenario like: 

`All pages > sub-page 1 > sub-page 2 > current page`

Where the pages' paths are not hierarchical: 

```
All pages: site.com
Sub-page 1: site.com/products
Sub-page 2: site.com/product/widget-api
Current page: site.com/product/widget-api/exampes
```

The above example is kind of like how you can go to `github.com/mapbox/dr-ui/pulls`, but PR 123 is at `github.com/mapbox/dr-ui/pull/123`. 

I can't think of anything in our subdomain like this and can't imagine we would really need to verify if a particular path follows some predetermined format. I guess what I was getting at was asking if you thought we would need any kind of validation that the `current site` path included all of its parent path. This 🦆 session has gone on too long now 😂

Reply:
Lol! Thanks for clarifying. I see what you're saying and our new systems of passing an array of links should support this!